userspace/libsinsp/sinsp.cpp

```diff
@@ -924,7 +924,11 @@ void sinsp::on_new_entry_from_proc(void* context,
 
 		if(!sinsp_tinfo)
 		{
-			ASSERT(tinfo != NULL);
+			if (tinfo == NULL)
+			{
+				// we have an fd but no associated tinfo, skip it
+				return;
+			}
```

Comment:
This looks fine, I'm just wondering if we're not doing something to the tune of ignoring all fds from scap files now :)

Reply:
It seems we are not doing any worse than before (at least comparing the output with a previous libs version)